python/lsst/ts/xml/data/sal_interfaces/LEDProjector/LEDProjector_Commands.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsl"?>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+  <SALCommand>
+    <Subsystem>LEDProjector</Subsystem>
+    <EFDB_Topic>LEDProjector_command_turnAllLEDsOn</EFDB_Topic>
```

Comment:
Consider dropping the "LED" from the command name, also, consider using "switch" instead, e.g. `LEDProjector_command_switchAllOn`.